src/entity.ts

```diff
@@ -271,6 +285,29 @@ export namespace entity {
           ]);
         },
       });
+
+      // `serialized` is computed on demand to consider any changes that may
+      // have been made to the key.
+      /**
+       * @name Key#serialized
+       * @type {array}
+       */
+      Object.defineProperty(this, 'serialized', {
```

Comment:
Why use `Object.defineProperty` as opposed to a regular getter?

Reply:
Give it a few years-- `Object.defineProperty` will be in style again, and the kids will be wearing it on their hoodies.